### ISSUE TITLE
`export default from 'module'` to `export { default } from 'module'` to 

### ISSUE THREAD

**Issue by mike1808** [benmosher/eslint-plugin-import#1042]
Shouldn't we have a rule forbidding exporting `default` export from `module` using old syntax?

```js
// good
export { default } from './module';

// bad 
export default from './module';
```

**Comment by ljharb** [comment]
Absolutely not; this isnt “old” syntax, both flavors are oart of ES2015.

If anything, we’d want to ban the named syntax: defaults are not just another name, another thing a module *has* - they’re what a module *is*. The special/different syntax is the best and proper thing to use.

**Comment by mike1808** [comment]
@ljharb I cannot find any mentioning of `export default from` in [ES2015](https://www.ecma-international.org/ecma-262/6.0/#sec-exports) not in the latest [draft](https://tc39.github.io/ecma262/#prod-ExportClause).

**Comment by ljharb** [comment]
ohh sorry, i missed the “from”. That’s also not old syntax, it’s new syntax - it’s a current stage 1 [proposal](https://github.com/tc39/proposal-export-default-from). Once it lands, it will be the preferred mechanism - using curly braces for default is the antipattern.

**Comment by mike1808** [comment]
@ljharb well then. My bad for calling it "old" :slightly_smiling_face:
